schema.graphql

```diff
@@ -51,4 +52,9 @@ type ValidatorExposure @jsonField {
 type IndividualExposure @jsonField {
   who: String!
   value: String!
-}
\ No newline at end of file
+}
+
+type Bonded @jsonField {
+  stash: String!
+  amount: String!
```

Comment:
Don't we plan to support several types of stake changes? Maybe it should be named with something more generic like StakeChange?